client/i18n/locales/swahili/motivation.json

```diff
@@ -101,7 +101,14 @@
     "Cowabunga!",
     "Moon Prism Power!",
     "Plus Ultra!",
-    "Everything's coming up Milhouse!"
+    "Everything's coming up Milhouse!",
```

Comment:
It looks like you accidentally edited the wrong file.

Please revert the changes back to its original state here.

This is the correct english file that should be updated

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/locales/english/motivation.json

Reply:
ok thank you for the kind help it should be alright now